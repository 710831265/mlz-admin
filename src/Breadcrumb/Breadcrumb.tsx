import React, { useState, useContext } from 'react';
import { default as AntdBreadcrumb } from 'antd/es/breadcrumb';
import 'antd/es/breadcrumb/style/index.less';

const Breadcrumb = AntdBreadcrumb;

export default Breadcrumb;
