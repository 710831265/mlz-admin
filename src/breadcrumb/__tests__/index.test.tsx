import React, { Component } from 'react';
import testMount from '../../../tests/testMount';
import testSnapshot from '../../../tests/testSnapshot';
import Breadcrumb from '..';

describe('ðŸ§ª Breadcrumb', () => {
  testMount(Breadcrumb);
  testSnapshot(Breadcrumb);
});
