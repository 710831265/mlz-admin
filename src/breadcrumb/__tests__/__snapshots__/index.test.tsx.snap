// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ðŸ§ª Breadcrumb snapshot test snapshot comparing 1`] = `"<div class=\\"ant-breadcrumb\\"></div>"`;
